### ISSUE TITLE
migrate to Zoe platform, JavaScript

### ISSUE THREAD

**Issue by dckc** [Agoric/eng-process#6]
@Tartuffo this sort of python stuff is typical of what I wrote for much / most of my career. But OCaps are ultimately necessary for cooperation without vulnerability, and the python community has decided not to go there:

> The python language doesn't enforce encapsulation. Moreover, the [We are all consenting adults](https://python-guide-chinese.readthedocs.io/zh_CN/latest/writing/style.html#we-are-all-consenting-adults) norm means that [OCap Discipline](https://github.com/dckc/awesome-ocap/wiki/OCap-Discipline) is not likely to become widely adopted. -- [DisciplinedPython · awesome\-ocap Wiki](https://github.com/dckc/awesome-ocap/wiki/DisciplinedPython)

So I am making the transition to JavaScript. Here's hoping we get to talk about migrating this project to JavaScript in due course. By way of example, see  https://github.com/dckc/ag-admin#readme

**Comment by Tartuffo** [comment]
FWIW, 1% of the code in agoric-sdk is written in Python: https://github.com/Agoric/agoric-sdk/search?l=python&p=1

**Comment by dckc** [comment]
yup; like you and me, warner has a lot more experience with python than javascript, so for rapid prototyping and data exploration, that's the tool he reaches for.

**Comment by erights** [comment]
> this sort of python stuff is typical of what...

From this bug report by itself, I have no idea what "this sort" refers to. ?

**Comment by dckc** [comment]
"this sort" refers to the code in https://github.com/Agoric/eng-process/tree/master/planning/issues
